extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-slider.js

```diff
@@ -156,5 +159,50 @@ describes.realWin(
         .querySelector('.i-amphtml-story-interactive-slider-bubble');
       expect(sliderBubble.textContent).to.be.equal('😄');
     });
+
+    it('should mock a true response and show post-selection state', async () => {
```

Comment:
Nit: "should show post-selection state when backend replies with user selection"